src/test/java/cd/go/contrib/elasticagents/dockerswarm/elasticagent/BaseTest.java

```diff
@@ -79,6 +80,21 @@ private static void removeSecrets() throws Exception {
         }
     }
 
+    private static void removeVolume() throws Exception {
+        if (dockerApiVersionAtLeast(docker, "1.26")) {
```

Comment:
Probably log this as a warning and any other place where we're testing for docker api version?